src/Kathara/cli/ui/setting/KubernetesOptionsHandler.py

```diff
@@ -153,7 +153,44 @@ def add_items(self, current_menu: ConsoleMenu, menu_formatter: MenuFormatBuilder
 
         image_pull_policy_item = SubmenuItem(image_pull_policy_string, image_pull_policy_menu, current_menu)
 
+        # Private Registry dockerconfigjson Option
+        pr_dcj_string = "Private Registry Docker Configuration"
+        pr_dcj_menu = SelectionMenu(
+            strings=[],
+            title=pr_dcj_string,
+            prologue_text="""Add a base64-encoded Docker configuration string to """
+                          """access private container registries. It will be used """
+                          """to create a Kubernetes secret of type `kubernetes.io/dockerconfigjson`.
+
+                          Default is %s.""" % DEFAULTS['private_registry_dockerconfigjson'],
+            formatter=menu_formatter
+        )
+
+        pr_dcj_menu.append_item(
+            FunctionItem(
+                text=pr_dcj_string,
+                function=setting_utils.read_value,
+                args=['private_registry_dockerconfigjson',
+                      RegexValidator(r'.*'),
+                      'Write a base64-encoded Docker configuration string:',

```

Comment:
I don't like the fact that the user have to insert a base64 string as value.
Can we enable the insertion of a path to the `docker.json` file and we open/encode it, writing the final value?

This would require some validation on the file, if it's a JSON and if it contains the right keys.

After encoding, we don't need the path anymore, and we just store the base64 value.

Reply:
Yes, this is a better solution. 
Could be useful to use the default path `~/.docker/config.json` and allow the user to specify a custom one.
After this we open and validate the file and store only the encoded base64 string as you said.